src/platform/imx8/include/platform/lib/memory.h

```diff
@@ -101,6 +101,7 @@
 #define HEAP_RT_COUNT256	32
 #define HEAP_RT_COUNT512	4
 #define HEAP_RT_COUNT1024	4
+#define HEAP_RT_COUNT2048	4
```

Comment:
You don't include it in the total size calculation on line 128.

Reply:
That only affected the debug (heap_show_...) or was it a deeper issue?

Reply:
Probably just debug.

Reply:
@paulstelian97 Actually it was more serious. You could potentially overwrite the next heap to the wrong base address calculated.